agent/eni/gonetwrapper/gonetwrapper_windows.go

```diff
@@ -0,0 +1,39 @@
+// +build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package gonetwrapper
```

Comment:
Can this whole package be implemented similar to `netlinkwrapper` ? 

Refer https://github.com/aws/amazon-ecs-agent/blob/master/agent/eni/netlinkwrapper/netlink_linux.go

Reply:
In essence, I have tried to implement it similar to networkutils while overcoming any shortcomings of former.
This package and netlinkwrapper are just wrappers over the underlying package.